src/cai_lab_to_nwb/zaki_2024/interfaces/miniscope_imaging_interface.py

```diff
@@ -267,22 +334,13 @@ def get_metadata_schema(self) -> dict:
         return metadata_schema
 
     def get_original_timestamps(self) -> np.ndarray:
-
-        timestamps_file_path = self.miniscope_folder / "timeStamps.csv"
-        assert timestamps_file_path.exists(), f"Miniscope timestamps file not found in {self.miniscope_folder}"
-
-        import pandas as pd
-
-        timetsamps_df = pd.read_csv(timestamps_file_path)
-        timestamps_milliseconds = timetsamps_df["Time Stamp (ms)"].values.astype(float)
-        timestamps_seconds = timestamps_milliseconds / 1000.0
-
+        timestamps_seconds = get_miniscope_timestamps(miniscope_folder_path=self.miniscope_folder)
         # Shift when the first timestamp is negative
-        # TODO: Figure why, I copied from miniscope
+        # TODO: Figure why, I copied from Miniscope. Need to shift also session_start_time
```

Comment:
It was part od the TODO. If we saved the session start time from the metadata, and the timestamps refer to that date. When we shift the timestamps to zero we need to shift back the session start time as well. But I left it there as a comment because I am not sure how to handle this. I will open a follow up PR only on this